### PR TITLE
(feature) add database context

### PR DESCRIPTION
**Pull request by marekbaranowski98**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/igroup-opole/yesterday-api/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by marekbaranowski98** [comment]
[[backend] Database Context](https://trello.com/c/FdIbX3sh/24-backend-database-context)

**Comment by rafalschmidt97** [comment]
Commits and pull request name doesn't follow our convention @marekbaranowski98

File: YesterdayApi/Utilities/Database/DatabaseExtension.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using DbContext = System.Data.Entity.DbContext;
+
+namespace YesterdayApi.Utilities.Database
+{
+  public static class DatabaseExtension
+  {
+    public static void AddDatabase(this IServiceCollection services)
+    {
+      services.AddDbContext<YesterdayContext>(opt =>
+        opt.UseSqlServer("Yesterday"));
```
Comment (fabiankoziolek) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]:
```suggestion
        opt.UseSqlServer("Connection string to our database in docker"));
```

Comment (AlexTorianyk) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]:
```suggestion
        opt.UseSqlServer(configuration.GetConnectionString("Yesterday")));
```

Comment (AlexTorianyk) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]:
```suggestion
        opt.UseSqlServer(configuration.GetConnectionString("Yesterday"));
```

File: YesterdayApi/Utilities/Database/DatabaseExtension.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using DbContext = System.Data.Entity.DbContext;
+
+namespace YesterdayApi.Utilities.Database
```
Comment (fabiankoziolek) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]: Database related stuff should be in infrastructure not in utilities

Comment (fabiankoziolek) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]: Database related stuff should be in infrastructure not in utilities

File: YesterdayApi/Utilities/Database/DatabaseExtension.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using DbContext = System.Data.Entity.DbContext;
```
Comment (fabiankoziolek) [inline YesterdayApi/Utilities/Database/DatabaseExtension.cs]: What is the purpouse of this line?

Comment (marekbaranowski98) [reply YesterdayApi/Utilities/Database/DatabaseExtension.cs]: I forgot to remove unnecessary lines

**Pull request closed without merging**